### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by cubap**
Start the testing harness for TinyThings in Node with Jest.

### CODE REVIEW

**Comment by cubap** [comment]
FYI, includes your `bryan-testing` branch

File: tokens.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 require('dotenv').config()
+import fetch from 'node-fetch'
```
Comment (cubap) [inline tokens.js:2]: something here is bad news bears and I had to comment it out for the moment in app.js

Comment (thehabes) [reply tokens.js:2]: Yeah I noticed.  That's OK, we will figure it out.  It's a deep cut Node thing.

**Review by thehabes — APPROVED** [review]
The only thing I noticed is that you are using `got` for all your fetch moments.  Is that something you hope we continue to use in the future?  I am not using `got` in my code, for example.  I am just using `fetch()`es.

**Comment by cubap** [comment]
yeah. `got` works and `node-fetch` is causing some grief right now. I may get rid of that in the `tokens.js` to prevent trouble.

**Pull request merged**